std/datetime.d

```diff
@@ -16153,6 +16153,21 @@ public:
         Params:
             duration = The duration to add to or subtract from this
                        $(LREF DateTime).
+
+        See_Also:
+            $(CXREF time, Duration).
```

Comment:
Suggest instead to change Param specification to:

The $(CXREF time, Duration) to add or subtract...